### PR TITLE
task/GraphQL-Basic : Completed Task

### PR DESCRIPTION
**Pull request by Skmnkn**
Task GraphQL Basic

Link [Task GraphQL Basic] (https://docs.google.com/document/d/1JFNrbzTj03DPHLANU_SXRf6eg5ENRC8g9V37WRlW9Tw/edit)

- [x] Add database in hasura with name `kampus_merdeka`
- [x] Add table anggota with `id` (*primarykey*, autoIncerement) and `nama`
- [x] Add table keterangan with `id` (*primarykey*, autoIncerement), `id_anggota`, `pelajaran`, `nilai`, `status` (*boolean*)
- [x] Make Query to call `status = true` 
- [x] Insert Data with Mutation for Angga
- [x] Update Data with Mutation where `status = false` set to `status = true` 
- [x] Added Screenshots Task
- [ ] Added Summary

### CODE REVIEW

File: 20_GraphQL-Basic/task-graphql-basic/01_callQueryStatusTrue/callQueryStatusTrue.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+query MyQuery {
+    kampus_merdeka_keterangan(where: {status: {_eq: true}}) {
+      id
+      id_anggota
+      nilai
+      pelajaran
+      status
+    }
+  }
```
Comment (syukronarie) [inline 20_GraphQL-Basic/task-graphql-basic/01_callQueryStatusTrue/callQueryStatusTrue.json]: seharusnya file ini formatnya .graphql

File: 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    insert_kampus_merdeka_keterangan(objects: {id_anggota: $id_anggota, pelajaran: $pelajaran, nilai: $nilai, status: $status}) {
+      returning {
+        id
+        id_anggota
+        pelajaran
+        nilai
+        status
+      }
+    }
+  }
```
Comment (syukronarie) [inline 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json]: seharusnya file ini formatnya .graphql

File: 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "nilai": 90,
+    "status": true
+  }
+
+  {
+    "id_anggota": 3,
+    "pelajaran": "React",
+    "nilai": 55,
+    "status": false
+  }
```
Comment (syukronarie) [inline 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json]: seharusnya ini dipisah dg format .json, format json selalu hanya dlm satu object

**Review by syukronarie — APPROVED** [review]

File: 20_GraphQL-Basic/task-graphql-basic/01_callQueryStatusTrue/callQueryStatusTrue.json (hunk 1 above)
Comment (Skmnkn) [reply 20_GraphQL-Basic/task-graphql-basic/01_callQueryStatusTrue/callQueryStatusTrue.json]: done mas, sudah saya perbaiki di commit selanjutnya.

File: 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json (hunk 2 above)
Comment (Skmnkn) [reply 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json]: done mas, sudah saya perbaiki di commit selanjutnya.

File: 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json (hunk 3 above)
Comment (Skmnkn) [reply 20_GraphQL-Basic/task-graphql-basic/02_InsertAnggawithMutation/InsertAnggawithMutation.json]: done mas, sudah saya pisah bagian .graphql dan .json, terimakasih mas

**Pull request merged**